package.json

```diff
@@ -71,12 +75,13 @@
     "nyc": "^13.0.0",
     "sinon": "^7.0.0",
     "source-map-support": "^0.5.6",
+    "tmp": "0.0.33",
     "ts-mockito": "^2.2.5",
     "typescript": "~3.2.0"
   },
   "files": [
     "out/src",
-    "out/third_party/types",
+    "out/third_party/cloud-debug-nodejs",
```

Comment:
The "cloud-debug-nodejs" part doesn't look right as we don't have that third_party dep anymore?

Reply:
There's still a third_party dependency. 
scanner.ts from cloud-debug-nodejs (which we still use in the source mapper) was copied without modification (it's a subtree), so it's in third_party.